### PR TITLE
Basic styling (Header / Footer)

### PR DESCRIPTION
**Pull request by Ofir-Smuha**
- Header & Footer added to relevant pages
- Margins and general appearance fixed

### CODE REVIEW

**Comment by morsdyce** [comment]
Need to fix conflicts

**Comment by Ofir-Smuha** [comment]
> Need to fix conflicts

Fixed.

File: src/components/common/Footer.js [hunk 1]
```diff
@@ -10,27 +10,25 @@ type Props = {
...
+        <CopyRights>2018 github, Inc.</CopyRights>
+        <Link>Terms</Link>
+        <Link>Privacy</Link>
+        <Link>Security</Link>
+        <Link>Status</Link>
+        <Link>Help</Link>
+      </LinksContainer>
+      <Icon onClick={() => props.history.push('/')} />
+      <LinksContainer>
+        <Link>Concatct GitHub</Link>
```
Comment (morsdyce) [inline src/components/common/Footer.js:24]: Got a typo in Contact

File: src/components/issues/details/IssueDetails.js [hunk 2]
```diff
@@ -73,17 +74,18 @@ class IssueDetails extends Component<ConnectedProps & OwnProps, OwnState> {
...
-        <ContentContainer>
-          <IssueContent
-            currentIssue={this.props.currentIssue}
-            issueComments={this.props.issueComments}
-            handleCommentSubmit={this.handleCommentSubmit}
-          />
-          <SideBar currentIssue={this.props.currentIssue} />
-        </ContentContainer>
-      </Wrapper>
+      <GlobalLayout userInfo={this.props.userInfo}>
```
Comment (morsdyce) [inline src/components/issues/details/IssueDetails.js]: GlobalLayout should be independent. It should not be aware of the user. Only the part that is relevant to displaying user data should be aware of it.

This also leads to the header not showing until the user is loaded which is an unpleasant jump in the page.

**Review by morsdyce — CHANGES_REQUESTED** [review]

File: src/components/issues/new-issue/NewIssue.js [hunk 3]
```diff
@@ -72,7 +71,4 @@ const Wrapper = styled.div`
   justify-content: space-between;
 `;
 
-const mapStateToProps = (state: StateType) => ({
-  userInfo: state.user.userInfo
-});
-export default withRouter(connect(mapStateToProps, { addNewIssue })(NewIssue));
+export default withRouter(connect(null, { addNewIssue })(NewIssue));
```
Comment (morsdyce) [inline src/components/issues/new-issue/NewIssue.js:74]: no need to change now but when you have multiple HoC you can use compose function from redux to write it in a way that's more readable. 

e.g:
```js
export default compose(
 withRouter,
 connect(null, { addNewIssue })
)(NewIssue)
```

**Review by morsdyce — APPROVED** [review]

**Pull request merged**